torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3311,6 +3311,23 @@ def generator():
     return list(generator())
 
 
+def sample_inputs_select(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
+
+    cases = (((S, S, S), (1, 2)),
+             ((S, S, S), (-1, 2)),
+             ((S, S, S), (-1, -1)),
+             ((S, S, S), (1, -1)),
+             ((S,), (0, 2))
+             )
```

Comment:
Similar to https://github.com/pytorch/pytorch/pull/56122#discussion_r613794404 have directly added the cases.